c++/src/kj/compat/gzip.h

```diff
@@ -29,12 +29,14 @@ KJ_BEGIN_HEADER
 
 namespace kj {
 
+#define KJ_GZ_BUF_SIZE 4096
+
 namespace _ {  // private
 
 class GzipOutputContext final {
 public:
   GzipOutputContext(kj::Maybe<int> compressionLevel);
-  ~GzipOutputContext() noexcept(false);
+  ~GzipOutputContext();
```

Comment:
Why did you remove `noexcept(false)` here? Per KJ style, all destructors should have this annotation.

https://github.com/capnproto/capnproto/blob/master/style-guide.md#exceptions-usage

Reply:
Adjusted it – I wasn't aware that the style guide requires `noexcept(false)`, I assumed it wasn't needed here as the destructor only calls a C cleanup function.